Data/Migrations/PuzzleServerContextModelSnapshot.cs

```diff
@@ -15,7 +15,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "2.1.8-servicing-32085")
+                .HasAnnotation("ProductVersion", "2.1.1-rtm-30846")

```

Comment:
It seems like we should leave the ProductVersion unchanged.  But maybe this doesn't matter?